packages/lesswrong/components/tagging/FilterMode.tsx

```diff
@@ -127,7 +121,34 @@ const FilterModeRawComponent = ({tagId="", label, mode, canRemove=false, onChang
     </span>
   </span>
 
-  const otherValue = isCustomFilterMode(currentUser, mode) ? (mode || "") : "";
+  const [inputTime, setInputTime] = useState(0);
+
+  const handleCustomInput = (input: string) => {
```

Comment:
Maaaybe instead of a timeout just wait for an onBlur? I'm not sure but it might make the code simpler and be a better user experience. OTOH, maybe just ship it.

Reply:
My one worry about that it that `onBlur` often occurs here when the user closes the popup rather than clicking outside the input, and I feel like it's a little odd to change the selected state whilst the popup is closed. If it's ok with you, I think I'll leave it as it is, but I'll change it at the first sign of a user complaint.

Reply:
Yep, seems good.